recipes/conversation_view/feed_pill/feed_item_pill.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <div
+    :class="[wrapperClass, expanded ? 'dt-feed-item--expanded-pill' : 'd-bar-pill']"
+  >
+    <div :class="['d-bgc-secondary', expanded ? 'dt-feed-item--expanded-pill' : 'd-bar-pill']">
+      <div class="d-p8">
+        <button
+          :aria-label="ariaLabel"
+          data-qa="dt-feed-item-pill"
+          class="
+            d-baw0
+            d-bgc-moderate
+            d-bar-pill
+            d-w100p
+            d-c-pointer
+            d-ta-left"
+          @focusin="hover = true"
+          @focusout="hover = false"
+          @mouseenter="hover = true"
+          @mouseleave="hover = false"
+          @click="$emit('click')"
+        >
+          <dt-item-layout class="d-w100p d-p8">
+            {{ title }}
+            <template slot="left">
+              <!-- TODO: Add transition styling to the icon on hover -->
```

Comment:
There's a click animation inside the prototype 
https://www.figma.com/proto/8ib3sY0ZVuBMGoxxceG6Ja/Ai-Tiger-Team?page-id=4186%3A130594&type=design&node-id=4186-134058&viewport=313%2C369%2C0.54&t=4myGHU3w75xvGK2d-1&scaling=min-zoom&starting-point-node-id=4186%3A134058&show-proto-sidebar=1

Might take a crack at this if there's time, but if not then I think we're ok with out it for now